ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -530,7 +533,7 @@ private void WriteModelFile(ModelRoot exportedModel, string filePath)
 
             var settings = new WriteSettings
             {
-                ImageWriting = ResourceWriteMode.SatelliteFile,
+                ImageWriting = SatelliteImages ? ResourceWriteMode.SatelliteFile : ResourceWriteMode.BufferView,
                 ImageWriteCallback = ImageWriteCallback,
```

Comment:
Just for correctness sake, is ImageWriteCallback called when using ResourceWriteMode.BufferView?

Reply:
It's called in `ResourceWriteMode.SatelliteFile` mode only.